scripts/Muon/GUI/Common/muon_group_pair_context.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import (absolute_import, division, print_function)
+
+import Muon.GUI.Common.utilities.xml_utils as xml_utils
+from Muon.GUI.Common.muon_group import MuonGroup
+from Muon.GUI.Common.muon_pair import MuonPair
+
+from mantid.api import WorkspaceGroup
+from mantid.kernel import ConfigServiceImpl
+from Muon.GUI.Common.observer_pattern import Observable
+
+
```

Comment:
tests for this?

Reply:
https://github.com/mantidproject/mantid/blob/a0dd9776ceac4f2ca557ed89193a60c0ee3cc64c/scripts/test/Muon/muon_group_pair_context_test.py